ui/partials/main/thGlobalTopNavPanel.html

```diff
@@ -75,6 +75,18 @@
                title="Treeherder help" href="help.html" target="_blank">
                 <span class="fa fa-question-circle lightgray nav-help-icon"></span></a>
 
+            <!-- Compactness Button -->
+            <span class="btn btn-view-nav btn-right-navbar nav-menu-btn"
+                  title="Toggle terse mode"
+                  ng-click="toggleCompactMode()"
+                  tabindex="0"><span>Compact</span>
+                <span class="label label-default"
```

Comment:
I think we're using 2 space indent for htm, but yeah I see this file is largely already using 4.